presto-common/src/test/java/com/facebook/presto/common/TestRuntimeStats.java

```diff
@@ -29,7 +29,9 @@ public class TestRuntimeStats
     private static final String TEST_METRIC_NAME_3 = "test3";
     private static final String TEST_METRIC_NAME_NANO_1 = "test_nano_1";
     private static final String TEST_METRIC_NAME_NANO_2 = "test_nano_2";
+    private static final String TEST_SHOULD_LESS_THAN_1_SECOND_DUMMYP_JOB_METRIC_NANO_TIME = "test_dummy_job_less_1_second";
```

Comment:
DUMMYP in the name ? is it a typo ?
why not have a simple name like `TEST_METRIC_NAME_NANO_3` ?

Reply:
I intended to explain the meaning of this metric in a way that doesn't cause confusion, but I'm fine using easier name like you suggested. 

Done, thanks!